docs/sources/flow/tasks/send-profiles-to-pyroscope/index.md

```diff
@@ -0,0 +1,133 @@
+---
+aliases:
+- /docs/grafana-cloud/agent/flow/tasks/send-profiles-to-pyroscope/
+canonical: https://grafana.com/docs/agent/latest/flow/send-profiles-to-pyroscope/
+description: Send profiles to Pyroscope 
+title: Send profiles Pyroscope 
+weight: 120
+---
+
+# Send profiles to Pyroscope
+
+Learn how to configure {{< param "PRODUCT_NAME" >}} to collect [Pyroscope Server][] profiles and forward them to a [Pyroscope Server][].
+
+This topic describes how to:
+
+* Setup [Pyroscope Server][].
+* Setup {{< param "PRODUCT_NAME" >}} to collect profiles and send them to [Pyroscope Server][].
+
+## Components used in this topic
+
+* [pyroscope.write][]
+* [pyroscope.scrape][]
+
+## Before you begin
+
+* Download {{< param "PRODUCT_NAME" >}}.
+* Install [docker][] if it is not already installed.
+
+## Steps
+
+1. Setup Pyroscope
+     
+   The easiest path to setting up Pyroscope is using `docker run -it -p 4040:4040 grafana/pyroscope`. This will spin up a docker instance running Pyroscope on port `4040`. The server will autoscrape itself and after a few minutes data will appear. If you immedietly access the server it may error with `No applications available` until it has scraped itself.  
+   
+   ![Initial Pyroscope server screen](../../../assets/tasks/initial-pyro.png)
+
+2. Download the [latest][] version of {{< param "PRODUCT_NAME" >}} for the system your are running. 
+
+3. Add the below configuration to `agent.river` file in the same directory as the file downloaded above.
```

Comment:
```suggestion
3. Add the following configuration to the `agent.river` file in the same directory as the file downloaded above.
```

I'm confused by this... you download the latest Agent... then save the River config file to the same directory as the downloaded Agent?  Why? I would expect to add the config file to the existing config or to the default config location. Am I misunderstanding what's oing on here?

Reply:
There is no default config location, `run PATH` must always be specified. With the services we do encode that into the service definition. There is no requirement to be in the same directory, but if we assume it is then it makes the commands more copy pastable.

Reply:
OK. The wording here still doesn't make sense to me.  maybe it'll resolve itself if this is reworked as a task. If we keep or expand it as a tutorial we'd need to fix this up and use the path either as a replaceable variable or explicitly name a location to use for the tutorial.